ElastosBountyProgram/front-end/src/config/data.js

```diff
@@ -375,4 +375,9 @@ export default {
         female: 'Female',
         other: 'Other',
     },
+    mappingRoleToName: {
+        MEMBER: 'Member',
```

Comment:
To be used as display values, those should be in the I18N module, and accessed from there for translation purposes. (consider ```I18N.get('key')```)